addons/popochiu/engine/objects/character/popochiu_character.gd

```diff
@@ -820,8 +823,7 @@ func _get_valid_oriented_animation(animation_label):
 		var animation = "%s%s" % [animation_label, suffix]
 		if $AnimationPlayer.has_animation(animation):
 			return animation
-	# No valid animation is found.
-	printerr('Animation not found %s' % [animation_label])
```

Comment:
This was probably intended as useful information for the developer, but I agree an error is too much.
Do you think we can log this info somewhere so that the dev has a log of missing animations that the engine expects to be played?

Reply:
I agree, but we already have a message that gives the proper context to devs. You can see an example of it in the screenshot that is part of the answer to your previous comment. I think there is no need to print this message if we already have a better one: it uses the Popochiu print style and gives a better feedback.